pkg/controller/serviceaccount/tokens_controller.go

```diff
@@ -120,6 +138,20 @@ func NewTokensController(serviceAccounts informers.ServiceAccountInformer, secre
 		options.SecretResync,
 	)
 
+	if e.legacyTokenCleanUp && e.legacyTokenCleanUpPeriod != 0 {
+		e.configMapInformer = informers.NewFilteredConfigMapInformer(
```

Comment:
I would expect this indexed informer to be passed to the controller init func as an argument